docs/api.md

```diff
@@ -142,6 +142,13 @@ package.
 const resp = await fetch('https://medv.io')
 ```
 
+We've attached a minor adjustment to the `fetch` API to make it more zx-friendly in pipe chains:
```

Comment:
Lets add why this may be needed. What disadvantages. Can a 1TB file be processed this way.